test/unit-tests/function/matrix/count.test.js

```diff
@@ -0,0 +1,34 @@
+// test count
+import assert from 'assert'
+
+import math from '../../../../src/defaultInstance.js'
+
+describe('count', function () {
+  function assertCount (count) {
+    assert.strictEqual(count([[1, 2, 3], [4, 5, 6]]), 6)
+    assert.strictEqual(count([[[1, 2], [3, 4]], [[5, 6], [7, 8]]]), 8)
+    assert.strictEqual(count([]), 0)
+    assert.strictEqual(count([[]]), 0)
+
+    assert.strictEqual(count('123456'), 6)
+    assert.strictEqual(count(''), 0)
+  }
+
+  it('should count something with a size as Matrix', function () {
+    assertCount(math.count)
+  })
+
+  it('should count something with a size as Array', function () {
+    const math2 = math.create({ matrix: 'Array' })
```

Comment:
This doesn't work as you expect: the `matrix` config only applies to matrices created when parsing an expression. 

Can you create some straightforward units instead which explicitly create a DenseMatrix and SparseMatrix? Like

```
assert.strictEqual(count(matrix([[1, 2, 3], [4, 5, 6]])), 6)

assert.strictEqual(count(sparse([[1, 2, 3], [4, 5, 6]])), 6)

```

Reply:
Oh... I was confused about the setting in `size()`. But it's only applied in the `string` case there

Reply:
👍